File: Client/src/app/Shared/Models/session-details-response.ts
export interface SessionDetailsResponse {
  id: string;
  patientName: string;
  PatientId:string;
  slotStart: string;
  slotEnd: string;
  durationInMinutes: number;
  status: number;
  editedTranscript: string;
  attachments: Attachment[];
  audioFileUrl: string | null;
  treatmentPlan: string | null;
}

export interface Attachment {
  id: string;
  fileUrl: string;
  fileName: string;
  fileType: string;
}

File: PhysioAssist.Api/Modules/SessionModule/Contracts/SessionDetailsResponse.cs
namespace PhysioAssist.Api.Modules.SessionModule.Contracts
{
    public class SessionDetailsResponse
    {
        public Guid Id { get; set; }
        public string PatientName { get; set; } = string.Empty;
        public DateTime? SlotStart { get; set; }
        public DateTime? SlotEnd { get; set; }
        public int DurationInMinutes { get; set; }
        public SessionStatus Status { get; set; }
        public string? EditedTranscript { get; set; }
        public string? AudioFileUrl { get; set; }

        public List<SessionAttachmentResponse> Attachments { get; set; } = [];
    }
}
